src/containers/MultidisciplinarySubject/components/MultidisciplinarySubjectArticle.tsx

```diff
@@ -139,6 +149,12 @@ const MultidisciplinarySubjectArticle = ({ topic, subject, resourceTypes, skipTo
       ? article.copyright.creators
       : article.copyright?.processors;
 
+  const licenseProps = licenseAttributes(
+    article.copyright?.license?.license,
+    article.language,
+    `${config.ndlaFrontendDomain}/${pathname}`,
```

Comment:
Om jeg tolker CC REL-guiden riktig skal vel dette bare være en tom string, i og med at vi refererer til nåværende side? https://opensource.creativecommons.org/ccrel-guide/.

Dette gjelder vel de andre stedene denne er brukt også

Reply:
Bruker vi egentlig noe særlig cc-attributter i den faktiske artikkelen, eller gjør vi det bare i "regler for bruk"? Kunne vi sluppet unna å bruke dette her?

Reply:
Jeg stusset litt på det, men mtp at vi faktisk viser lisensen på slutten av artikkelen så syns jeg at det blir mest riktig å ha det med her også. Jeg har lagt til cc:attributionName på navnet til forfatteren. Det blir litt dobbelt opp å ha det i artikkelen og i "Sitere eller gjenbruke" men det blir mest riktig?

Og når det gjelder `about=""` så ser det ut som jeg tolket det litt feil. Vi trenger ikke engang å ha det som en tom string, for det var inkludert i eksempel bare for _"completeness"_. Det gjør vel ingen skade å ha det sånn heller, men kan fjerne hvis vi vil slippe par kodelinjer

Reply:
Tenker det er like greit å ikke ha det isåfall. Å linke til seg selv blir litt merkelig, synes jeg.

Reply:
Da fjerner jeg det 👍